app/javascript/application.js

```diff
@@ -1 +1,2 @@
 import "./controllers";
+import "@hotwired/turbo-rails";
```

Comment:
If you pull from `main` and resolve the conflicts, I think this will already be included